### PR TITLE
Checkbox hit area

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #1497.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add a charm to the canvas
- open the unit list in the inspector
- clicking just outside the checkboxes should toggle the checkbox state
- clicking near the unit name should navigate to the unit details

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7698/

File: jujugui/static/gui/src/app/components/check-list-item/test-check-list-item.js [hunk 1]
```diff
@@ -178,16 +186,43 @@ describe('CheckListItem', () => {
...
+    output = renderer.getRenderOutput();
+    const expected = (
+      <input
+        disabled={false}
+        type="checkbox"
+        id="a-label-item"
+        onClick={instance._stopBubble}
+        onChange={instance._handleChange}
+        checked={true} />);
+    expect(output.props.children.props.children[0].props.children).toEqualJSX(
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/check-list-item/test-check-list-item.js]: Maybe assign to variables to make the whole assertion more readable?

Comment (Lukewh) [inline jujugui/static/gui/src/app/components/check-list-item/test-check-list-item.js]: I agree with when paths/ subsets are used multiple times.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7705/

**Comment by huwshimi** [comment]
Thanks @frankban and @Lukewh :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**